pkg/controllers/user/helm/common/common.go

```diff
@@ -82,13 +115,15 @@ func GenerateRandomPort() string {
 func InstallCharts(rootDir, port string, obj *v3.App) error {
 	setValues := []string{}
 	if obj.Spec.Answers != nil {
+		IsSystemCatalog(obj)
```

Comment:
This function should be call outside of `if obj.Spec.Answers != nil {`

Reply:
I don't think so, you know, if it is called outside of `if obj.Spec.Answers != nil {`, then the app will be installed if users have set `system-default-registry` yet not launched logging.

Reply:
Yeah, updated!